pom.xml

```diff
@@ -85,6 +85,19 @@
         <project.reporting.outputEncoding>utf-8</project.reporting.outputEncoding>
         <surefire.version>2.19.1</surefire.version>
     </properties>
+    <repositories>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=xml%3AIndentCheck)